### DISCUSSION TITLE
How to resovle error :failed to find group named

### DISCUSSION THREAD

**Discussion by hatzhang** [fluxcd/flux2#1142] [Q&A]
My gitlab server is set up using HyperV linux vm on internal server (192.168.10.120). And my test env is also a hyperv linux vm on my laptop. When i try to ```flux bootstrap``` with my personal gitlab token, i got the following error.
```
► connecting to 192.168.10.120
✗ failed to get project, error: failed to find group named 'MyRepoGroup'
```
But with the same settings on server linux vm, it went well. At first i thought there maybe some network issue, but i can ```git clone``` the repo or ```ssh``` into server. So what should i do to resolve this?

**Comment by hiddeco** [comment]
This is probably solved in #968. I have added instructions to the PR description to build the binary on your machine, it would be awesome if you can confirm this indeed solves the issue you are experiencing. :sparkles:

**Comment by hatzhang** [comment]
@hiddeco Because of unstable network, can't verify this. Is there any other workaround? try some old version of fluxcli?
